pkg/vpcmodel/drawioGenerator.go

```diff
@@ -106,6 +106,11 @@ func (g *groupedExternalNodes) GenerateDrawioTreeNode(gen *DrawioGenerator) draw
 	name := "Various IP ranges"
 	if all, _ := isEntirePublicInternetRange(*g); all {
 		name = publicInternetNodeName
+	} else {
+		_, ipBlock := g.toIPBlocks()
+		if len(ipBlock.ToCidrList()) == 1 {
```

Comment:
How about the case where we have a single IP range that cannot be represented as a single CIDR?
Can we show the IP range instead of "Various IP ranges"?

Reply:
yes, done